plugins/parodos/src/components/workflow/WorkflowsOverview.tsx

```diff
@@ -111,6 +119,13 @@ export function WorkflowsOverview(): JSX.Element {
             margin="none"
           />
         </Grid>
+        <Grid item xs={5} md={3} lg={2} className={classes.mtaReportLinkItem}>
+          <Link target="_blank" href="#">
```

Comment:
```suggestion
          <Link target="_blank" href="#" rel="noopener noreferrer">
```

Reply:
If the link is cross-origin add `rel="noopener noreferrer"`.

[source](https://developer.chrome.com/en/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/)